src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  onTabSelected,
+  selectedTabId,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) ?? tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+
+          {tabs.map(tab => (
+            <li
+              className={cn({ 'is-active': selectedTab.id === tab.id })}
```

Comment:
This check can be placed in a separate variable and reused for the `onClick` check.